plugins/eos-blacklist/gs-plugin-eos-blacklist.c

```diff
@@ -207,11 +226,30 @@ app_get_flatpak_ref_name (GsApp *app)
 	return gs_app_get_metadata_item (app, "flatpak::RefName");
 }
 
+static gchar **
+gs_plugin_get_default_locales (GsPlugin *plugin)
```

Comment:
What was the reasoning for choosing the name `gs_plugin_get_default_locales()`? Typically, the prefix `gs_plugin_` is only used in plugins for the plugin vfuncs which they implement. Internal helper functions should be named differently.

I’d call this `get_default_locales()`.